about.html

```diff
@@ -0,0 +1,142 @@
+            </div>
+            <div class="container">
+                <h1 class="logo"><a href="./index.html"><img src="./assets/alogo/wdh0.svg" alt="web hero logo"
+                            width="200px"></a></h1>
+                <ul>
+                    <li class="tab-active"><a class="mlink" href="about.html">About</a></li>
+                    <li><a class="mlink" href="./index.html#cources">Cources</a></li>
+                    <li><a class="mlink" href="./index.html#mentors">Mentors</a></li>
+                    <li><a class="mlink" href="./index.html#partners">Partners</a></li>
+                    <li class="tab-active"><a class="mlink Campaign" href="">Register Now!</a></li>
+                </ul>
+            </div>
+        </nav>
+        <div class="softnav">
+            <a href="./index.html"><i class="fa fa-home"></i>Home</a> <b class="tab-active">></b> <a href="#">about </a>
+        </div>
+    </header>
+
+    <main class=".main">
+        <section class="hero">
+            <img class="a2" src="assets/rand/a2.png">
+            <img class="a3" src="assets/rand/a3.png">
+
+
+            <div class="container">
+                <div class="text-main">
+                    <p>"Hello! hero developers"</p>
+                    <h1>WEB-HERO-CAMP <br> HERO DEVS IN 2023</h1>
+
+                </div>
+                <div class="para">
+                    <p>This your base where you start your journey towords becoming a true hero! A hero who does code
+                        like a super power. Power to bring ones vision to life. With robustness, speed, scalability and
+                        all the best practices one can have, you develop like a hero. A super hero fullstack developer
+                        in the world of web.</p>
+                </div>
+                <div class="nextdate">
+                    <p>Please contact us per Email for any further questions about Web Dev Code Camp 2023!
+                        <br><u><b>webherocamp@info.com</b></u>
+                    </p>
+                </div>
+            </div>
+        </section>
+
+        <section class="mentors">
+            <h1>Web Hero Developer Code Camp 2023 Logo</h1>
+            <div class="line"></div>
+            <p>The logo of Web Hero Fullstack Web Developer Code Camp 2023 was decided through the logo competition from
+                20, Jan to 01 Feb</p>
+            <p class="loremtxt">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ut ullam exercitationem
+                deserunt corporis dignissimos sequi dolorum! Voluptatem, exercitationem facilis? Blanditiis sunt
+                accusantium quisquam maxime odio veniam mollitia doloribus eum. Aliquid laborum ipsum eum ullam
+                perferendis quidem modi tenetur dignissimos numquam? Cumque odio aut rerum impedit aliquam molestias
+                distinctio iure pariatur.</p>
+            <img src="assets/alogo/whlogo.png" alt="">
```

Comment:
- [ ] Please ensure that every `img` element on your website contains an `alt` attribute :+1:. This attribute provides a brief, descriptive text alternative for users who may be unable to view the images due to slow network connections, assistive technology, or any other reason. Including the `alt` attribute not only helps to improve the accessibility of your website but also benefits search engine optimization (SEO) by providing more context to search engines about the images on your site :+1:.

Reply:
Thanks a lot for this indepth review. Really appriciate it. Will addess each and make necessary changes.